### ISSUE TITLE
FNX-5203 ⁃ Update product name

### ISSUE THREAD

**Issue by mheubusch** [mozilla-mobile/fenix#1714]
As a user, I want to see the Firefox Preview name and logo throughout the app experience so that I know I am using the correct product. 

### Acceptance Criteria (Added by PM)
I can see Firefox Preview in the  GPS
I can see Firefox Preview or a smaller version of the name beneath the app icon on my Android home screen
I can see Firefox Preview logo lock up on the app home screen
I can see the Firefox Preview name throughout the application
I can not see the Fenix name in the application

**Comment by sblatz** [comment]
I will need an updated image for this home view asset with both a light `#FBFBFE` and dark text `#20123A`
<img width="350" alt="image" src="https://user-images.githubusercontent.com/4400286/56377241-d3680180-61be-11e9-9544-1049332e7039.png">

**Comment by vesta0** [comment]
@mheubusch FYI :)

**Comment by shorlander** [comment]
![firefox-preview-horizontal@4x](https://user-images.githubusercontent.com/684427/56394595-d2a68e00-6205-11e9-9003-5c9ab92ca2bb.png)

![firefox-preview-horizontal-ondark@4x](https://user-images.githubusercontent.com/684427/56394618-e7832180-6205-11e9-8b59-b6882b25e78d.png)

**Comment by shorlander** [comment]
[ic_launcher.zip](https://github.com/mozilla-mobile/fenix/files/3096430/ic_launcher.zip)

**Comment by bifleming** [comment]
@sblatz is this still blocked?

**Comment by shorlander** [comment]
![firefox-preview-stacked@4x](https://user-images.githubusercontent.com/684427/56436752-f079fe80-62aa-11e9-8e9e-73a9d9cd03d9.png)
![firefox-preview-stacked-ondark@4x](https://user-images.githubusercontent.com/684427/56436754-f2dc5880-62aa-11e9-8922-2f39e0fe75ec.png)

**Comment by sblatz** [comment]
Thanks for the updated icons, @shorlander!

I'm labeling this blocked again until I have the "debug" icon. Shorlander mentioned he was working on it 😄

**Comment by shorlander** [comment]
[ic_launcher-nightly.zip](https://github.com/mozilla-mobile/fenix/files/3104700/ic_launcher-nightly.zip)

**Comment by sv-ohorvath** [comment]
@sblatz On Google Play, with a user subscribed to the Fenix google group to receive builds, I'm still seeing the Fenix app, with no logo. Can't find a Fenix Preview app.

**Comment by gaby2300** [comment]
I can reproduce this using a Sony Xperia Z3C with Android 6.0.1. The home icon is an unnamed dark gray square. Opening the app, I see the Firefox Preview icon at the top.

**Comment by sblatz** [comment]
I believe this fix didn't hit nightly. Are you able to test off of our master build? @sv-ohorvath

**Comment by sv-ohorvath** [comment]
> I believe this fix didn't hit nightly as it merged just 30 minutes ago. Are you able to test off of our master build? @sv-ohorvath
> 
> Either that or it'll be in nightly tomorrow :)

I can see the new logo and app name since the 4.24 build. I can't build locally atm, I'm getting build errors.

**Comment by sblatz** [comment]
@sv-ohorvath Sorry, got confused on which ticket this was. This should be working on newer device, but there was an issue that was [fixed just an hour ago](https://github.com/mozilla-mobile/fenix/issues/1970) for older devices. 

What device are you having issues with and what icon(s) can you not see?

**Comment by sv-ohorvath** [comment]
@sblatz I was looking on Google Play (as a subscribed user) for the updated product, but I'm seeing the old Fenix product name without a logo. I was expecting to get Firefox preview in there as well. But maybe I'm wrong?
Device: Pixel (Android 9)
![Screenshot_20190425-164618](https://user-images.githubusercontent.com/4037709/56746940-cc745c80-6785-11e9-9668-eb8cf448c920.png)

**Comment by yoasif** [comment]
I see  the same thing as @sv-ohorvath on a Pixel 2.

**Comment by kbrosnan** [comment]
Someone from the Release Management team @sylvestre? would need to update the assets in Google Play. When that is done please add back the QA needed label so we can verify and close.

**Comment by sylvestre** [comment]
Yeah, this is on us, please follow: https://wiki.mozilla.org/Release_Management/Adding_a_new_app_on_Google_play#Distribution

**Comment by kbrosnan** [comment]
@shorlander are you able to provide the assets or redirect?

**Comment by shorlander** [comment]
> @shorlander are you able to provide the assets or redirect?

I can provide the Hi-Res icon, should loop in @lwa-moz to discuss screenshots and feature images.

**Comment by mheubusch** [comment]
@shorlander Have you provided the hi-res icon already?  I removed the Needs UX label here - is that OK?

**Comment by sblatz** [comment]
I think we're still waiting on the icon for this, @mheubusch, @shorlander.

**Comment by shorlander** [comment]
> I think we're still waiting on the icon for this, @mheubusch, @shorlander.

Which icons are still needed?

**Comment by sblatz** [comment]
@shorlander we need a google play store asset I believe? @sylvestre is that correct?

**Comment by vesta0** [comment]
I believe @shorlander and @lwa-moz own the play store assets.

**Comment by shorlander** [comment]
![icon-google-play-store](https://user-images.githubusercontent.com/684427/57412778-f3885080-71bf-11e9-8bc8-2498a5e8ab77.png)
Play Store image
@sblatz

**Comment by sblatz** [comment]
I have pinged @mitchhentges to take action on the rel eng side 😄

**Comment by mitchhentges** [comment]
Hey all :wave: I can't update the Google Play Store, but @s.ylvestre (on PTO atm) is your guy :)

**Comment by sylvestre** [comment]
@sblatz done for the two!

**Comment by vesta0** [comment]
Thank you @sylvestre !

**Comment by sv-ohorvath** [comment]
> @sblatz I was looking on Google Play (as a subscribed user) for the updated product, but I'm seeing the old Fenix product name without a logo. 
I can see the new logo, but unfortunately, the product name is not updated yet: still seeing "Fenix" in the Play store.

**Comment by sylvestre** [comment]
@sv-ohorvath I didn't change it, what should I enter?

**Comment by sv-ohorvath** [comment]
@sylvestre The new product name is Firefox Preview
> ### Acceptance Criteria (Added by PM)
> I can see Firefox Preview in the GPS

**Comment by sylvestre** [comment]
updated

**Comment by gaby2300** [comment]
I received the new update. Google Play shows Firefox Preview. However, the
home icon is still Fenix.

**Comment by kbrosnan** [comment]
On some older devices the name or icon may not update when we make these sorts of changes. Sometimes removing the icon from the home screen then adding it back from the app drawer fixes the problem. Otherwise you would need to uninstall and reinstall via Google Play (this will delete all local data including bookmark and passwords). 

We have made this change before we released so all new users will get the correct name and icon.
